src/content/docs/alerts-applied-intelligence/transition-guide.mdx

```diff
@@ -1,15 +1,79 @@
 tags:
   - Alerts and Applied Intelligence
   - New Relic Alerts
   - EOL
   - Transition guide
 metaDescription: 'Resources to make a successful transition as new functionalities in New Relic One replace alerting functionalities reaching end of life in 2022.'
+redirects:
+  - /docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/alerts-ai-transition-guide-2022 
 ---
 
 Over the course of several months during 2022, new functionalities in New Relic One will replace alerting functionalities reaching end of life (EOL). We are providing these early notices so you can plan accordingly and make a smooth transition.
 
+## What's changed [#changed]
+
+Starting in late 2021, we've reorganized the navigation UI for alerts and applied intelligence. Although the alerts and applied intelligence navigation has changed pretty dramatically, all the features you use are still there.
+
+We're slowly rolling out this update to all of our users.
+
+![A screenshot of the new alerts and applied intelligence left navigation.](./images/alerts-ai-categories.png "The new alerts and applied intelligence left navigation")
+
+<figcaption>The new UI includes these new categories.</figcaption>
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="nav-changes"
+    title="An overview of alerts and applied intelligence nav changes"
+  >
+
+  ## Analyze [#analyze]
+
+The [overview page](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page) shows your most recent open, closed, and muted violations.
+
+The **Issues & activity** page shows your most recent [issues](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#issues), [incidents](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#incidents), and [anomalies](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#anomalies).
+
+## Detect [#detect]
+
+The [alert conditions (policies) page](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/) lists the policies where your conditions are created and organized. Create new policies here.
+
+The [anomaly detection page](/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/expanded-anomaly-detection/) lists your expanded anomaly detection configurations. You can add or edit the configurations here.
+
+## Correlate [#correlate]
+
+The [sources page](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/#1-configure-sources) shows your configured incident intelligence sources. You can add new sources or configure existing ones.
+
+The [decisions page](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions/) shows your correlated incidents. You can review, edit, and add decisions.
+
+## Enrich & notify [#enrich]
```

Comment:
not sure you want to use an ampersand even though if it's in the UI...?
But I'm not quite understanding what "enrich" means here. For me, the newbie, adding muting rules would do exactly the opposite. Maybe point out the reason why muting rules are a happy thing to enrich (zero in on?) the things you care about so you can focus on something else? Like, you could set up muting rules when testing something but you want them to be on for the go-live moment, or for specific holiday shopping timeframes, or...?

Reply:
Muting rules are more on the notify side of things, than the enrich side of things. Personally, I feel like muting rules should go under settings :D 

Yeah, I don't like the ampersand there either, but I'm erring on the side of matching the UI, I guess.

I'm hoping that my intro sentence will help clarify this a bit.